src/main.rs

```diff
@@ -1,159 +1,226 @@
-    let namespace_info = query_api_namespace(token.to_string(), endpoint.to_string(), namespace.to_string())?;
+    println!(
+        "{}",
+        format!("\nQuerying API for namespace {}...", namespace).to_string()
+    );
+    let namespace_info = query_api_namespace(
+        token.to_string(),
+        endpoint.to_string(),
+        namespace.to_string(),
+    )?;
     print!("\nAPI Response: ");
-    println!("{} {:?} {}", namespace_info.name, namespace_info.admins, namespace_info.last_deployment);
-   
-    // Query the DB and get back a table of already added namespaces 
+    println!(
+        "{} {:?} {}",
+        namespace_info.name, namespace_info.admins, namespace_info.last_deployment
+    );
+
+    // Query the DB and get back a table of already added namespaces
     let current_table: Vec<DBItem> = get_db_namespace_table()?;
     println!("\nCurrent Table of Projects:");
     let mut db_table = Table::new(); // Create the table
     db_table.add_row(row!["Namespace", "Admins", "Latest Deployment"]); // Add a row per time
     for row in &current_table {
-        db_table.add_row(row![row.name, format!("{:?}", row.admins), row.last_deployment]);
+        db_table.add_row(row![
+            row.name,
+            format!("{:?}", row.admins),
+            row.last_deployment
+        ]);
     }
     db_table.printstd(); // Print the table to stdout
 
-    // Check if the namespace queried for is in the DB, and if not, ask to put it in. 
-    if !current_table.iter().any(|x| x.name.to_string() == namespace_info.name.to_string()) {
+    // Check if the namespace queried for is in the DB, and if not, ask to put it in.
+    if !current_table
+        .iter()
+        .any(|x| x.name.to_string() == namespace_info.name.to_string())
+    {
         println!("\nThis namespace is not in the database! Would you like to add it? (y/n): ");
         let mut input = String::new();
-        io::stdin().read_line(&mut input).expect("Could not read response");
+        io::stdin()
+            .read_line(&mut input)
+            .expect("Could not read response");
         if input.trim() == "y" {
             println!("Putting a ShelfLife on {}", namespace_info.name.to_string());
             let _table_add = add_item_to_db_namespace_table(namespace_info);
-        }else if input.trim() == "n" { println!("Ok."); } else { println!("Invalid response."); }
-    } else { println!("The requested namespace is in the database.") }
+        } else if input.trim() == "n" {
+            println!("Ok.");
+        } else {
+            println!("Invalid response.");
+        }
+    } else {
+        println!("The requested namespace is in the database.")
+    }
     Ok(())
 }
 
-
-fn query_api_namespace(token: String, endpoint: String, namespace: String) -> Result<DBItem, Box<std::error::Error>> {
+fn query_api_namespace(
+    token: String,
+    endpoint: String,
+    namespace: String,
+) -> Result<DBItem> {
     let client = reqwest::Client::new();
```

Comment:
This is good. Since you have multiple types called `Client`, it's good to use the fully-qualified name to distinguish them (like you did here with `reqwest::Client`) :+1:

Reply:
Huh. Full names only. Got it.

Reply:
Not always, just in this case because the short names for both of these is `Client`